Tests not running docker images:  ERROR: for drupal  Container "97e3d9afc3b2" is unhealthy

Our testing workflow seems to have stopped running since Saturday.


See: https://github.com/localgovdrupal/localgov_project/actions

![image](https://github.com/localgovdrupal/drupal-container/assets/326588/e01128cf-cb89-42cc-95fb-be02e43885dd)

Seems to be failing with "ERROR: for drupal  Container "97e3d9afc3b2" is unhealthy"

@andybroomfield noticed this on localgov_alert_banner and @Adnan-cds reported the tests were running on Saturday.


```

Run docker-compose -f docker-compose.yml up -d
Creating network "localgov_project_default" with the default driver
Pulling database (mariadb:latest)...
latest: Pulling from library/mariadb
Digest: sha256:dd51b32c5c5c6ed56019bb92f48b4f[7](https://github.com/localgovdrupal/localgov_project/actions/runs/5240567962/jobs/9461597582#step:4:8)492[8](https://github.com/localgovdrupal/localgov_project/actions/runs/5240567962/jobs/9461597582#step:4:9)7208b1b[9](https://github.com/localgovdrupal/localgov_project/actions/runs/5240567962/jobs/9461597582#step:4:10)03ac61ef1efa6c2ae24[10](https://github.com/localgovdrupal/localgov_project/actions/runs/5240567962/jobs/9461597582#step:4:11)
Status: Downloaded newer image for mariadb:latest
Pulling chromedriver (selenium/standalone-chrome:106.0)...
106.0: Pulling from selenium/standalone-chrome
Digest: sha256:5e9c5548a1e7e8e0b5[14](https://github.com/localgovdrupal/localgov_project/actions/runs/5240567962/jobs/9461597582#step:4:15)24f24ac6abd746edbd98c07983c793ee50c597e5ccbd
Status: Downloaded newer image for selenium/standalone-chrome:106.0
Pulling drupal (localgovdrupal/apache-php:php8.1)...
php8.1: Pulling from localgovdrupal/apache-php
Digest: sha256:d921cbe7a3250414293f9851d24bbafe7552fdf5525c4896c4c3abdc1fd53fb5
Status: Downloaded newer image for localgovdrupal/apache-php:php8.1
Creating database ... 
Creating chromedriver ... 
Creating chromedriver ... done
Creating database     ... done

ERROR: for drupal  Container "97e3d9afc3b2" is unhealthy.
Encountered errors while bringing up the project.
Error: Process completed with exit code 1.
```

I've just tried locally and managed to build the docker containers and run tests OK.

@andybroomfield wondered if this might be related to Docker Hub's changes for open source projects.

@stephen-cox any thoughts?

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -39,7 +39,7 @@ services:
       MARIADB_USER: database
       MARIADB_PASSWORD: database
     healthcheck:
-      test: ["CMD", "mysqladmin" ,"ping", "-h", "localhost"]
+      test: ["CMD", "/usr/local/bin/healthcheck.sh", "--connect"]
       timeout: 5s
       retries: 100
 

